docs/Documentation/Applications/ansys.md

```diff
@@ -0,0 +1,82 @@
+#Using ANSYS on Eagle
+
+ANSYS is a licensed commercial application for the Eagle system. 
```

Comment:
We decided on putting the page description in italics. (See github planning meeting chat)